### ISSUE TITLE
code question

### ISSUE THREAD

**Issue by yuyijie1995** [dongdonghy/repulsion-loss-faster-rcnn-pytorch#9]
`if int(torch.sum(gt[j]==gt[z]))==4:
                        overlaps[j,z]=0`
I do not understand why we need it, can you give me some explanation?
